portfolio/src/main/webapp/index.html

```diff
@@ -37,6 +37,14 @@ <h2>Comments</h2>
           <input type="submit"/>
         </form>
       <div id="text-container"></div>
+      <label for="comment-number">Maximum number of comments to display on one page: </label>
+      <select name="comment-number" id="number" onchange="showComments()">
```

Comment:
The ID is not very specific - could you please choose a name which indicates better what this is?